### DISCUSSION TITLE
【脑洞】艺术化公链

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#36] [Ideas]
刚才跟 @Ungigdu 的讨论

[可以建真的区块链呀](https://github.com/qihuju/qihuju.github.io/discussions/25#discussioncomment-597585)

我的一个脑洞是：区块画框

> 1，“区块画框”是一个隐喻，其实就是对“智能合约”的“修辞修饰”，类似“商业模式画布”

> 2，之所以叫做“区块画框”，是希望吸引“创作类”的人群关注智能合约和区块链

> 3，“区块画框”的推广模式会采用curating（策展）的玩法，会比一般区块链项目的推广更有趣。类似beeple拍卖事件之于NFT

这个脑洞我在一个多月之前分别跟 @askender @MingxuanXie @inkoml 讨论过，但是没有持续下去
这个脑洞远远还没有成熟到建立一个project，或者一个理事会，就当成一个脑洞吧，看看有没有人有兴趣讨论下去……

**Comment by Sleeper2060** [comment]
这个project的潜力很大，玩法也可以很多。可能需要更多人的参与和密集的讨论会。project前的闲聊(brain storm)很重要。哈哈哈。

**Reply by Lupingyue** [reply]
>   这个project的潜力很大，玩法也可以很多。可能需要更多人的参与和密集的讨论会。project前的闲聊(brain storm)很重要。哈哈哈。

对呀，先脑洞起来。
随时……

> 等待更多人参与

> 密集的讨论会「可以是clubhouse内部房间」

> brain storm

**Comment by Ungigdu** [comment]
随时脑洞

**Comment by askender** [comment]
脑洞是一方面。我先说点务实的。
我自己也发过基于substract的pow公链（外界可访问但没宣传和运营），发公链的门槛不在于技术，而是合规性、运营实力、同质化竞争。
合规方面中国法律比较严和模糊（新加坡还不错）。
运营实力和创新度以及资本有关（艺术公链要么激励机制和玩法创新、版权创新比如nft、存储和分发创新比如存储币），纯pow的币也容易被利用（有抗asic限制cpu和gpu的但要么中心化要么安全不足）pos容易成为资本游戏但是常见的妥协、 由于没有预挖和ico新币的共识很难做大。
同质化竞争在于，公链要有足够的不可替代性。当前的脑洞其实并不包含。

我建议的具体做法（仅供参考）：
用ipfs/filecoin/ar等来做去中心化存储
用flow等来做nft，广义的版权和许可玩法，也可以用likecoin的iscn.io的方式去策划。
激励机制，我目前拥有大量likecoin的资源，打算用于空投和社区激励以及各种项目的grant
协议层和激励层用已有的项目，暂时不重复造轮子。  （你看flow团队不也是先用加密猫来探索找到痛点和技术团队积累才开发公链引发共识的么）
然后重心放在艺术内容实质的品质、应用层的玩法创新、社区的繁荣上。

在也许遥远也许很近的未来，公链和出版在法律世界更加自由和成熟，去中心化的生态和技术更成熟，发链才是更优的选择。   现在发玩具链的意义大家也可以讨论，我目前从乐观改为了悲观。

泼了冷水，然后再聊脑洞。
- 发公链的最大意义在于，足够充分的激励。那么机制设计一定要合理，否则不如利用现有的公链。机制脑洞在技术上暂时没有突破。岳老师的那个实践比较偏向于行为艺术，似乎没有包含激励机制的设计。
- 公链的第二个意义在于别的公链提供不了的玩法。这一点上期待大家的脑洞。我目前陷在去中心化基础设施、授权和激励机制等已有的玩法上了。
- 很期待应用层的脑洞。这种脑洞更容易借助已有的公链拿到grant实际做出来（我自己也在拿grant做一些项目）。我觉得这种脑洞意义也是挺大的。
- 说点具体的容易被认为是做广告，还是说一点。赞赏和激励的玩法可以继续挖掘，比CC&nft更有意思的许可机制我也在like&iscn中逐步推进实践，以后大家可以用无数种预定义的许可来增加分发、策展、赞赏的玩法。

**Reply by Lupingyue** [reply]
待会细读

**Reply by Ungigdu** [reply]
握个爪，同开发substrate

**Reply by askender** [reply]
握爪   你主要做哪一块呢？   我目前在filecoin、likecoin都有一些资源，有兴趣的话可以一起做。如果是nft的话，我也关注flow的进展。  波卡生态没有具体想借用的链， cosmos生态最近发现其实也渐渐有起色了。
我觉得找到可以借用的共识，拿grant先做起来会比较好。

**Reply by Ungigdu** [reply]
我之前一直是以太坊生态开发者，关注Dapp开发和用区块链解决实际问题，EIP-3135的作者是我。现在在弄一些小东西，考虑开发nft这样比较快的，技术难度小，重运营的玩法

**Reply by askender** [reply]
厉害！    现在第一个需要解决的是，「比较快的，技术难度小，重运营的玩法」是否独立做公链相比与借助现有的链是更好的选择。

**Comment by Lupingyue** [comment]
刚才在小群里跟 @Ungigdu 的讨论

“艺术化公链”是一个长期目标，我们可以先玩一个NFT测试一下

**哈希肖像|hash icon**

> **孙振宇 Deus ex machina:** 
> @岳路平 老板这么能搞事情，发个币呗
> 我给技术支持呀
> 你搞一个艺术家邀请制的小圈子，每邀请一个就发一个nft给他
> 

> **宝可梦:** 
> 发给他的这个nft是有“内容”的？还是没有内容的？
> 是像crypto punks那种吗

> **孙振宇 Deus ex machina:** 
> 有没有内容都行
> nft内容不就是一个uri吗
> uri是url的超集吧，你可以认为是url，就一外部链接，你想展示什么都行

> **宝可梦:** 
> 我在想不能简单只给一个nft，特别是给艺术家的话，应该让他们觉得“可以创作”，至少是可以“挑选”。如果可以给这个nft起个名字随机获得一个“面具”，这样艺术家就有“参与感”。这也是我在GitHub跟你聊的“区块画框”的其中一个意思

> **孙振宇 Deus ex machina:** 
> 嗯，有道理，nft的内容可以共创是个亮点，现在项目还没有好像
> 其实我可以用算法保证nft的uri对应的资源是nft持有者才能修改的，可以对应一个作画网站，一人一笔，可以传递下去什么的

> **宝可梦:** 
> 就叫“哈希肖像”，“hash icon”，这样还可以策展一个画展，让人觉得很艺术的样子
> 一开始不需要很复杂，只要小白都能操作就可以，万一跑通了再优化

> **孙振宇 Deus ex machina:** 
> 没事，一开始可以是个文档，你添一行我加一行。文档的前端是个页面，要求你进行签名，持有nft的人有可以签名的私钥就行。你考虑玩法吧，技术方面不是问题

**Reply by askender** [reply]
如果有国外的朋友代理、有启动资金、会长期开发会比较好。否则容易半途而废。否则借助现有的链其实也能玩一些花样。

**Reply by Sleeper2060** [reply]
宝可梦: 
我在想不能简单只给一个nft，特别是给艺术家的话，应该让他们觉得“可以创作”，至少是可以“挑选”。如果可以给这个nft起个名字随机获得一个“面具”，这样艺术家就有“参与感”。这也是我在GitHub跟你聊的“区块画框”的其中一个意思
孙振宇 Deus ex machina: 
嗯，有道理，nft的内容可以共创是个亮点，现在项目还没有好像
其实我可以用算法保证nft的uri对应的资源是nft持有者才能修改的，可以对应一个作画网站，一人一笔，可以传递下去什么的
宝可梦: 
就叫“哈希肖像”，“hash icon”，这样还可以策展一个画展，让人觉得很艺术的样子
一开始不需要很复杂，只要小白都能操作就可以，万一跑通了再优化
孙振宇 Deus ex machina: 
没事，一开始可以是个文档，你添一行我加一行。文档的前端是个页面，要求你进行签名，持有nft的人有可以签名的私钥就行。你考虑玩法吧，技术方面不是问题

😊跟我的project的idea很像，
现在就是我的这个project没有这么多的技术支持，导致不会这么精彩。
在考虑要不把我的project直接先用奇忽局启动，
我就可以多点时间放到这里来，
然后先跟少数的艺术家合作。
算是一个小的测试、小的启动。
技术开发已经有 @askender 孙振宇和 @MingxuanXie ,
我负责像艺术家推这些ideas。

**Reply by Lupingyue** [reply]
> 
> 😊跟我的project的idea很像，
> 现在就是我的这个project没有这么多的技术支持，导致不会这么精彩。
> 在考虑要不把我的project直接先用奇忽局启动，
> 我就可以多点时间放到这里来，
> 然后先跟少数的艺术家合作。
> 算是一个小的测试、小的启动。
> 技术开发已经有 @askender 孙振宇和 @MingxuanXie ,
> 我负责像艺术家推这些ideas。

好主意，支持！

**Reply by Ungigdu** [reply]
可以，找个时间线上讨论一下吧。github看的不及时
